x-pack/platform/plugins/shared/actions/server/routes/connector/create/create.ts

```diff
@@ -47,13 +48,18 @@ export const createConnectorRoute = (
     },
     router.handleLegacyErrors(
       verifyAccessAndContext(licenseState, async function (context, req, res) {
-        const actionsClient = (await context.actions).getActionsClient();
-        const action = transformCreateConnectorBodyV1(req.body);
-        return res.ok({
-          body: transformConnectorResponseV1(
-            await actionsClient.create({ action, options: req.params })
-          ),
-        });
+        try {
+          const actionsClient = (await context.actions).getActionsClient();
+          const action = transformCreateConnectorBodyV1(req.body);
+          const resp = await actionsClient.create({ action, options: req.params });
+          const body = transformConnectorResponseV1(resp);
+
+          return res.ok({
+            body,
+          });
+        } catch (error) {
+          return errorHandler(res, error);
```

Comment:
I just realized there is no test for this. Could you please test it?

Reply:
Added here d581c54029782638ace857cb0090804dc832a3f1